lib/database/models/dataPass.js

```diff
@@ -46,6 +46,11 @@ module.exports = (sequelize) => {
     DataPass.associate = (models) => {
         DataPass.belongsTo(models.LhcPeriod, { foreignKey: 'lhcPeriodId', as: 'lhcPeriod' });
         DataPass.belongsToMany(models.Run, { as: 'runs', foreignKey: 'data_pass_id', through: 'data_passes_runs' });
+        DataPass.belongsToMany(models.SimulationPass, {
+            as: 'simulationPass',
```

Comment:
I would name this `anchoredSimulationPasses` (plural at least)

Reply:
applied